### ISSUE TITLE
Add support to validate XML against XSD 1.1

### ISSUE THREAD

**Issue by mkordas** [jcabi/jcabi-xml#110]
Currently method `com.jcabi.xml.XSDDocument#validate` uses XSD 1.0 parser. XSD 1.1 has couple of new features added, like possibility to require exactly one attribute out of two that will be useful in other Teamed projects. 

I'd expect just to replace 1.0 parser with 1.1, as version 1.1 is just superset of 1.0.

**Comment by dmarkov** [comment]
@yegor256 please take a look at it and dispatch (see [par.21](http://at.teamed.io/policy.html#21))

**Comment by yegor256** [comment]
@mkordas but we're not using any specific parser. we're just using JDK. parser is provided in runtime. or I didn't get the problem?

**Comment by mkordas** [comment]
@yegor256 you are right! Do you think it is in scope of integration test, Javadoc or http://xml.jcabi.com/example-xsd.html to provide instruction how to enable XSD 1.1 parsing?

**Comment by yegor256** [comment]
@mkordas yes, let's document it

**Comment by dmarkov** [comment]
@mkordas thanks a lot for reporting, **15 mins** added to your acc, pmt ID `56af291dc2680807d7000398`

**Comment by mkordas** [comment]
@yegor256 actually I've realized that we may have to modify line https://github.com/jcabi/jcabi-xml/blob/master/src/main/java/com/jcabi/xml/XSDDocument.java#L158. We should set it to `http://www.w3.org/XML/XMLSchema/v1.1` according to https://xerces.apache.org/xerces2-j/faq-xs.html

**Comment by longtimeago** [comment]
@yegor256 can we proceed here?

**Comment by longtimeago** [comment]
@yegor256 could you please respond to messages above?

**Comment by longtimeago** [comment]
@yegor256 ping :)

**Comment by longtimeago** [comment]
@yegor256 please, proceed here :)

**Comment by yegor256** [comment]
@dmarkov pls assign @westonized to this ticket

**Comment by dmarkov** [comment]
> @dmarkov pls assign @westonized to this ticket

@yegor256 OK @westonized please proceed, this task is yours

**Comment by longtimeago** [comment]
@westonized any progress so far?
